com/win32com/test/testPyComTest.py

```diff
@@ -866,6 +868,35 @@ def TestQueryInterface(long_lived_server=0, iterations=5):
         tester.TestQueryInterface()
 
 
+def TestRememberedGeneratedTypeLibraries():
+    typelib_ob = pythoncom.LoadTypeLib(
+        os.path.join(
+            localWin32comDirectory.parent, "TestSources", "PyCOMTest", "PyCOMTest.tlb"
```

Comment:
I'm sure a different TypeLib that always exists on Windows can be used for this. So that this test doesn't have to depend on PyCOMTest